ext/symengine/symengine.c

```diff
@@ -145,4 +146,14 @@ void Init_symengine() {
     rb_define_module_function(m_symengine, "zeta", cfunction_zeta, 1);
     rb_define_module_function(m_symengine, "gamma", cfunction_gamma, 1); 
 
+    //NTheory Functions as Module Level Functions
+    rb_define_module_function(m_symengine, "gcd", cntheory_gcd, 2);
+    rb_define_module_function(m_symengine, "lcm", cntheory_lcm, 2);
+    rb_define_module_function(m_symengine, "nextprime", cntheory_nextprime, 1);
+    rb_define_module_function(m_symengine, "%", cntheory_mod, 2);
```

Comment:
Defining this as a module level function won't work. You are not going to use it like `SymEngine::%(5, 4)` right?